source/dub/dependency.d

```diff
@@ -698,7 +698,7 @@ struct Version {
 	}
 
 	/// Tests if this represents the special unknown version constant.
-	@property bool isUnknown() const { return m_version is UNKNOWN_VERS; }
+	@property bool isUnknown() const { return m_version == UNKNOWN_VERS; }
```

Comment:
You need to make `UNKNOWN_VERS` a `static immutable` otherwise it might fail

Reply:
It is.